### PR TITLE
Document build errors

### PR DESCRIPTION
**Pull request by mira01**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: MAINTENANCE.md [hunk 1]
```diff
@@ -8,6 +8,22 @@
...
+* Github release could not be created as the same release already exists.
+  Possible Solutions:
+   - Delete the release and retry.
+   - It's highly probable that also Pypi release exists, so it's easier to bump version and try with higher version tag
+* Pypi upload failed with file already exists.
+  The file with the same name ever existed, it does not matter that it no longer exists
+  (see [the linkded document](https://test.pypi.org/help/#file-name-reuse))
+  Possible solutions:
+  - It's desirable to have a release in consistent state (all packages of the given version updoaded in pypi),
+    so in case there was partial success, please upload rest of pasckages manually as described in
```
Comment (pcerny) [inline MAINTENANCE.md:22]: `pasckages` -> `packages`

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR63

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/56/ : SUCCESS in 1m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/56/ : SUCCESS in 1m 25s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/56/ : SUCCESS in 20s

**Pull request merged**